docs/advanced-features/static-html-export.md

```diff
@@ -25,7 +25,7 @@ Update your build script in `package.json` to use `next export`:
 }
 ```
 
-Running `npm run build` will generate an `out` directory.
+Running `npm run build` will generate an `.next` directory.
```

Comment:
The current documentation is correct as right above this line is `next build && next export` instead of just `next build`. It does output `.next` but that's not relevant to this part of the docs as it's about `next export` output.

Reply:
yes you're right. thanks